Updated For New Mutation Table

tanner-coon-bh opened the pull request:
Updated Schemas to reflect changes to data model. Updated MutationSqlGenerator* files to reflect changes to mutation table. Removed work-around code for quirks with somatic_mutation from Filter code.

tanner-coon-bh committed:
Updated For New Mutation Table

Updated Schemas to reflect changes to data model. Updated MutationSqlGenerator* files to reflect changes to mutation table. Removed work-around code for quirks with somatic_mutation from Filter code.

abradyGDIT commented on src/main/java/bio/terra/cda/app/generators/MutationSqlGenerator.java:
```diff
@@ -2,7 +2,8 @@
 
-@EntityGeneratorData(entity = "somatic_mutation", hasFiles = false, defaultOrderBy = "case_barcode",
+// TODO - case_barcode may need to be altered
+@EntityGeneratorData(entity = "mutation", hasFiles = false, defaultOrderBy = "case_barcode",
```
OPTIONAL: `defaultOrderBy = integer_id_alias` might save us a bit of sorting time

tanner-coon-bh replied:
Updated

abradyGDIT reviewed (dismissed):
approved including one highly optional suggestion

tanner-coon-bh committed:
Updated Mutation Count Endpoint

Added "one_consequence" column to the summarized mutation count endpoint.

tanner-coon-bh committed:
Updated Mutation Default Order By

Changed mutation default order by from case_barcode to integer_id_alias

tanner-coon-bh committed:
Updated expected query for CountSqlGeneratorTest

Updated expected test result to match the new optimized count query

tanner-coon-bh committed:
Bypassing CountSqlGeneratorTest

Auto passing the test because it currently isn't written to test against the optimized query
